pom.xml

```diff
@@ -154,22 +154,22 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.itsallcode</groupId>
-                <artifactId>openfasttrace-maven-plugin</artifactId>
```

Comment:
why is this not used anymore?

Reply:
Because it's broken and breaks the build too. I'll need to invest some time to fix it, but I'll do it after refactoring